src/editor/HexEditor.tsx

```diff
@@ -14,10 +15,25 @@ const HexEditorContainer = styled.div`
     background: #252526
 `
 
-export const HexEditor = () => (
-    <HexEditorContainer>
-        <OffsetPanel></OffsetPanel>
-        <HexPanel></HexPanel>
-        <TextPanel></TextPanel>
-    </HexEditorContainer>
-)
\ No newline at end of file
+const hexStringFormat = 16
+const bitsPerByte = 8
+const bitsCountToNexLine = 16
+
+const generateHexLines = (data:Buffer) => 
+    Array(Math.ceil(data || Buffer.from(' ') ? data.byteLength / bitsPerByte : 0))
```

Comment:
Can't we split this part into another function?

The Buffer.from(' ') is still obscure to me, maybe we can put is into a const as well?
0 is the default hex value?